ethclient/client.go

```diff
@@ -15,10 +15,8 @@ import (
 
 var ErrInvalidEndpoint = errors.New("invalid rpc endpoint")
 
-type Client ethclient.Client
```

Comment:
If we're removing the `Client` type here, should we rename the package/file? 

I think would better belong under `utils` now, since it is just a utility for creating a `subnet-evm/ethclient` instance with headers/query params.